ui/index.d.ts

```diff
@@ -176,6 +176,7 @@ export declare abstract class AzExtTreeItem {
     public description?: string;
     public iconPath?: TreeItemIconPath;
     public commandId?: string;
+    public tooltip?: string;
```

Comment:
Problematically this gets called earlier than is ideal. We could use `resolveTreeItem` to solve this issue.